src/Pages/miyoungshin/Main/Component/FeedList.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import CommentList from './CommentList';
+import COMMENT from '../Data/CommentData';
+import REACTION from '../Data/FeedReactionData';
+
+class HandleFeed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      commentsList: [],
+      commentValue: '',
+      isPostActive: false,
+      userId: 'wecode',
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      commentsList: COMMENT,
+    });
+  }
+
+  inputChange = e => {
+    this.setState({
+      commentValue: e.target.value,
+    });
+  };
+
+  postButtonChange = e => {
+    if (this.state.commentValue !== '') {
+      e.key === 'Enter'
+        ? this.addComment()
+        : this.setState({ isPostActive: true });
+    } else {
+      this.setState({ isPostActive: false });
+    }
+  };
```

Comment:
```suggestion
  postButtonChange = e => {
    if (this.state.commentValue === '') {
      this.setState({ isPostActive: false });
      return
    }
    
    if (e.key === 'Enter') {
      this.setState({ isPostActive: true });
      return
    }
    
    this.addComment()
  };
```
이 함수가 최종적으로 3가지의 시나리오를 갖고 있다는 걸 조금 더 명확히 하고, if문으로 인한 depth를 줄이기 위해 early return 방식을 사용하는게 어떨까요?

Reply:
수정해 보겠습니다ㅎㅎㅎㅎ